apps/condo/domains/contact/utils/clientCard.tsx

```diff
@@ -0,0 +1,145 @@
+import { Col, Row, Typography } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import { gql } from 'graphql-tag'
+import { NextRouter } from 'next/router'
+import qs from 'qs'
+import React from 'react'
+
+import Select from '@condo/domains/common/components/antd/Select'
+import { colors, fontSizes } from '@condo/domains/common/constants/style'
+import { TICKET_PROPERTY_FIELDS } from '@condo/domains/ticket/gql'
+import { getAddressRender } from '@condo/domains/ticket/utils/clientSchema/Renders'
+
+export enum ClientType {
+    Resident,
+    NotResident,
+}
+
+const SEARCH_BY_PHONE = gql`
+    query searchByPhone ($organizationId: ID, $phone: String) {
+        contacts: allContacts(
+          where: {
+            organization: { id: $organizationId },
+            phone_contains: $phone
+          }, first: 10) {
+            id
+            name
+            phone
+            property { ${TICKET_PROPERTY_FIELDS} }
+            unitName
+        }
+      
+        tickets: allTickets(
+          where: {
+            organization: { id: $organizationId },
+            clientPhone_contains: $phone,
+            isResidentTicket: false,
+            clientPhone_not: null,
+            clientName_not: null,
+        }, first: 10) {
+          id
+          property { ${TICKET_PROPERTY_FIELDS} }
+          clientName
+          clientPhone
+          unitName
+      }
+      
+        employees: allOrganizationEmployees(
+          where: {
+            organization: { id: $organizationId },
+            phone_contains: $phone,
+          }, first: 10) {
+            id
+            name
+            phone
+        }
+    }
+`
+
+async function _search (client, query, variables) {
+    return await client.query({
+        query: query,
+        variables: variables,
+        fetchPolicy: 'network-only',
+    })
+}
+
+export function searchByPhone (organizationId) {
+    if (!organizationId) return
+
+    return async function (client, phone) {
+        const { data, error } = await _search(client, SEARCH_BY_PHONE, { phone, organizationId })
+
+        if (error) console.warn(error)
+
+        const employees = []
+        const contacts = data.contacts.map(({ id, phone, property, unitName }) => ({
+            value: id, phone, property, unitName, type: ClientType.Resident,
+        }))
+        const tickets = data.tickets.filter(ticket => {
+            const employee = data.employees.find(employee => employee.phone === ticket.clientPhone)
```

Comment:
Why you filtering result by phone when GraphQL query already has filters?

Reply:
we need to find tickets from employees => such not resident tickets, where `clientPhone` === `employee.phone`